lib/internal/test_runner/reporter/dot.js

```diff
@@ -1,18 +1,24 @@
 'use strict';
 
+const colors = require('internal/util/colors');
 const {
   MathMax,
 } = primordials;
 
+const failedTestSymbol = '\u2716 ';
```

Comment:
This currently duplicates the same value as https://github.com/nodejs/node/blob/168178658e1ae980345e148d154d6e5516f93fa0/lib/internal/test_runner/reporter/spec.js#L24

Perhaps these should be extracted to some common `reporterUtils` file etc

Reply:
Do you mean a util file like colors.js for all the symbols?

Reply:
Something like that, but it should be in the scope of the reporters...